### PR TITLE
Move networking to common stack

### PR DESCRIPTION
**Pull request by andylangridge**
- Since we can only have 1 app service plan per VNet integration, moving to 1 VNet per environment. So I've put the networking resources in the common stack

### CODE REVIEW

File: app/stacks/appeals-service/terragrunt.hcl [hunk 1]
```diff
@@ -11,11 +11,17 @@ dependency "common" {
...
+    private_dns_zone_id              = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mock_resource_group/privateDnsZones/mock_id"
   }
 }
 
 inputs = {
   app_insights_connection_string   = dependency.common.outputs.app_insights_connection_string
   app_insights_instrumentation_key = dependency.common.outputs.app_insights_instrumentation_key
   app_service_plan_id              = dependency.common.outputs.app_service_plan_id
+  endpoint_subnet_id               = dependency.common.outputs.endpoint_subnet_id
+  integration_subnet_id            = dependency.common.outputs.endpoint_subnet_id
```
Comment (lestermarch) [inline app/stacks/appeals-service/terragrunt.hcl:25]: Should be var.integration_subnet_id?

File: app/stacks/applications-service/terragrunt.hcl [hunk 2]
```diff
@@ -11,11 +11,17 @@ dependency "common" {
...
+    private_dns_zone_id              = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mock_resource_group/privateDnsZones/mock_id"
   }
 }
 
 inputs = {
   app_insights_connection_string   = dependency.common.outputs.app_insights_connection_string
   app_insights_instrumentation_key = dependency.common.outputs.app_insights_instrumentation_key
   app_service_plan_id              = dependency.common.outputs.app_service_plan_id
+  endpoint_subnet_id               = dependency.common.outputs.endpoint_subnet_id
+  integration_subnet_id            = dependency.common.outputs.endpoint_subnet_id
```
Comment (lestermarch) [inline app/stacks/applications-service/terragrunt.hcl:25]: Should be integration_subnet_id?

**Review by lestermarch — APPROVED** [review]
See comments regarding integration_subnet_id in the terragrunt.hcl files.

File: app/stacks/appeals-service/terragrunt.hcl (hunk 1 above)
Comment (andylangridge) [reply app/stacks/appeals-service/terragrunt.hcl]: Fixed.

**Pull request merged**